tests/curators/test_curators_examples.py

```diff
@@ -214,9 +215,21 @@ def test_dataframe_curator(small_dataset1_schema: ln.Schema, ccaplog):
         assert str(error).startswith("column 'sample_note' not in dataframe")
     curator.standardize()
     curator.validate()
+
     artifact.delete(permanent=True)
 
 
+def test_dataframe_curator_index():
+    """Test validating a DataFrame index."""
+    df = datasets.small_dataset1(otype="DataFrame", with_index_type_mismatch=True)
+    schema = ln.Schema(index=ln.Feature(name="test", dtype="str").save()).save()
```

Comment:
Please delete the schema and the feature at the end of this test so that it doesn't interact with other tests.